src/formatDate.js

```diff
@@ -7,8 +7,43 @@
  *
  * @returns {string}
  */
-function formatDate(date, fromFormat, toFormat) {
-  // write code here
+function convertToYYYY(year) {
+  const numYear = parseInt(year, 10);
+
+  return numYear < 10
+    ? '200' + numYear
+    : numYear < 30
+      ? '20' + numYear
+      : '19' + numYear;
```

Comment:
The condition on line 13 is not necessary and is causing an incorrect conversion of years. The task states that if the year is less than 30, it should be prefixed with '20', and otherwise with '19'. This means that even if the year is less than 10, it should still be prefixed with '20'. So, you should remove the condition on line 13 and change the condition on line 15 to check if the year is less than 30.